src/ert/_c_wrappers/job_queue/run_status.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import dataclass
+from enum import Enum
+
+
+class StatusEnum(Enum):
+    RUNNING = "RUNNING"
+    FINISHED = "FINISHED"
+    CANCELED = "CANCELED"
+    FAILED = "FAILED"
+    NOT_STARTED = "NOT_STARTED"
```

Comment:
Its more common to use 

```python
from enum import Enum, auto

class StatusEnum(Enum):
  RUNNING = auto()
  FINISHED = auto()
  CANCELED = auto()
  FAILED = auto()
  NOT_STARTED = auto()
```
https://docs.python.org/3/library/enum.html#enum.auto